pr_check.sh

```diff
@@ -20,15 +20,24 @@ IQE_FILTER_EXPRESSION=""
 
 set -exv
 
+CHROME_SHA=$(curl -X GET "https://quay.io/api/v1/repository/cloudservices/insights-chrome-frontend/tag/" | jq .tags\[0\].name -r)
+CHROME_CONTAINER_NAME=chrome-$CHROME_SHA
+
+docker run -d --name $CHROME_CONTAINER_NAME --replace --network bridge quay.io/cloudservices/insights-chrome-frontend:$CHROME_SHA
+
```

Comment:
COpy the index after the run to dist and make sure dist exists

Reply:
@Hyperkid123 that was it!

TL;DR for the curious - this was not required in the landing page repo/`pr_check.sh` as the route to index.html for landing page existed on the proxied env (stage). However for rbac-ui this was not the case - and therefore we get 404's when redirected to `/index.html`.